Sources/X509/RDNAttribute.swift

```diff
@@ -381,123 +381,11 @@ extension RelativeDistinguishedName.Attribute: DERImplicitlyTaggable {
-            ///       utf8String        UTF8String
-            ///                           (SIZE (1..ub-organization-name)),
-            ///       bmpString         BMPString
-            ///                           (SIZE (1..ub-organization-name))  }
-            ///
-            ///
-            /// id-at-organizationalUnitName AttributeType ::= { id-at 11 }
-            ///
-            /// -- Naming attributes of type X520OrganizationalUnitName:
-            /// --   X520OrganizationalUnitName ::=
-            /// --          DirectoryName (SIZE (1..ub-organizational-unit-name))
-            /// --
-            /// -- Expanded to avoid parameterized type:
-            /// X520OrganizationalUnitName ::= CHOICE {
-            ///       teletexString     TeletexString
-            ///                           (SIZE (1..ub-organizational-unit-name)),
-            ///       printableString   PrintableString
-            ///                           (SIZE (1..ub-organizational-unit-name)),
-            ///       universalString   UniversalString
-            ///                           (SIZE (1..ub-organizational-unit-name)),
-            ///       utf8String        UTF8String
-            ///                           (SIZE (1..ub-organizational-unit-name)),
-            ///       bmpString         BMPString
-            ///                           (SIZE (1..ub-organizational-unit-name)) }
-            /// ```
-            case .RDNAttributeType.commonName,
-                .RDNAttributeType.localityName,
-                .RDNAttributeType.stateOrProvinceName,
-                .RDNAttributeType.organizationName,
-                .RDNAttributeType.organizationalUnitName:
-
-                switch valueNode.identifier {
-                case ASN1UTF8String.defaultIdentifier:
-                    value = try .init(utf8String: String(ASN1UTF8String(derEncoded: valueNode)))
-                case ASN1PrintableString.defaultIdentifier:
-                    value = try .init(printableString: String(ASN1PrintableString(derEncoded: valueNode)))
-                default:
-                    value = .init(storage: .any(ASN1Any(derEncoded: valueNode)))
-                }
-
-            /// ```
-            /// -- Naming attributes of type X520countryName (digraph from IS 3166)
-            ///
-            /// id-at-countryName       AttributeType ::= { id-at 6 }
-            ///
-            /// X520countryName ::=     PrintableString (SIZE (2))
-            /// ```
-            case .RDNAttributeType.countryName:
-                switch valueNode.identifier {
-                case ASN1PrintableString.defaultIdentifier:
-                    value = try .init(printableString: String(ASN1PrintableString(derEncoded: valueNode)))
-                default:
-                    value = .init(storage: .any(ASN1Any(derEncoded: valueNode)))
-                }
-
+            switch valueNode.identifier {
```

Comment:
Neat! I didn't know that this was allowed but it makes sense

Reply:
It's not _generally_ expected as many types don't preserve it, but it's allowed here.